### PR TITLE
update the docs for life after probin

### PR DESCRIPTION
**Pull request by zingale**
## PR checklist

- [ ] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

File: Docs/source/build_system.rst [hunk 1]
```diff
@@ -241,8 +241,8 @@ This is the current build system process.
 
   .. index:: write_probin.py
 
-  * This writes the Fortran module that holds the Microphysics runtime
-    parameters, ``extern.F90``.  This is output in
+  * This writes the routines that manager the Microphysics runtime
```
Comment (maxpkatz) [inline Docs/source/build_system.rst:244]: typo

File: Docs/source/creating_a_problem.rst [hunk 2]
```diff
@@ -58,12 +54,10 @@ Here:
...
-* `namelist` indicates if the variable should be in the namelist and
-  controlled at runtime.  The namelist column should have a ``y`` if
-  you want the parameter to be read from the ``&fortin`` namelist in
-  the ``probin`` file at runtime.  If it is empty or marked with
-  ``n``, then the variable will still be put into the
-  ``probdata_module`` but it will not be initialized via the namelist.
+* `namelist` indicates if the variable should be able to be set as
+  a runtime parameter.  If it is empty or marked with
+  ``n``, then the variable will still be creates in the ``problem`` namespace,
+  but it will not be able to be set via the commandline or inputs file.
```
Comment (maxpkatz) [inline Docs/source/creating_a_problem.rst:60]: We should probably add a sentence here to explain why you'd want to use this (i.e. for global variables you want to set at runtime based on some combination of the other parameters or other information about the problem setup).

File: Docs/source/inputs.rst [hunk 3]
```diff
@@ -2,70 +2,41 @@
...
-    (EOS, reaction network, ...)
 
-Only the ``inputs`` file is specified on the commandline. The
-associated ``probin`` file is specified in the inputs file
-using the ``amr.probin_file`` parameter, e.g.::
-
-    amr.probin_file = my_special_probin
-
-for example, has the Fortran code read a file called ``my_special_probin``.
+Only the ``inputs`` file is specified on the commandline.
```
Comment (maxpkatz) [inline Docs/source/inputs.rst:25]: I think this sentence looks a little odd without the context about probin *not* being set at the command line, maybe just delete it?

Comment (zingale) [reply Docs/source/inputs.rst:25]: right, I think the point I want to remind is that we need to set an inputs file on the commandline

Comment (maxpkatz) [reply Docs/source/inputs.rst:25]: That would be fine, maybe just remove the "only" then.

Comment (zingale) [reply Docs/source/inputs.rst:25]: well... I guess you can do it all as runtime parameters.  I'll remove the line.

File: Docs/source/creating_a_problem.rst (hunk 2 above)
Comment (zingale) [reply Docs/source/creating_a_problem.rst:60]: done

File: Docs/source/build_system.rst (hunk 1 above)
Comment (zingale) [reply Docs/source/build_system.rst]: fixed

**Review by maxpkatz — APPROVED** [review]

**Pull request merged**